sympy/geometry/ellipse.py

```diff
@@ -140,7 +140,9 @@ def __new__(
                 "eccentricity" must not be None.'''))
 
         if eccentricity is not None:
-            if hradius is None:
+            if simplify(eccentricity).is_negative == True:
```

Comment:
I'd drop simplify here

Reply:
It is raising error while using integer so `symplify` was added

Reply:
What was the problem with integer value?
I'd expect integers doesn't need simplify to pass the check correctly.

Reply:
@sylee957 
```
>>> from sympy import *
>>> x=5
>>> x.is_negative
```
Error raised
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'int' object has no attribute 'is_negative'
```
So symplify was added.

Reply:
Your problem looks like `sympify` than `simplify`

Reply:
@sylee957  Should I remove `simplify`?

Reply:
Yes, I’d encourage to remove simplify applied in assumption checks because it grows out as slow as the expression gets bigger, making computations impractical.

Reply:
Added `S` instead of `simplify`